algorithms/CPlusPlus/Graphs/prim-algorithm.cpp

```diff
@@ -0,0 +1,144 @@
+#include <limits.h>
+
+#include <iostream>
+#include <set>
+#include <vector>
+#include <map>
+
+#define MAXNUMVERTICES 25
+
+using namespace std;
+
+struct Edge
+{
+    int from, to, weight;
+    Edge();
```

Comment:
missing the brackets will cause an error
```suggestion
    Edge(){};
```